flocker_bb/builders/flocker.py

```diff
@@ -434,10 +441,85 @@ def makeOmnibusFactory(distribution, triggerSchedulers=()):
     return factory
 
 
+HOMEBREW_RECIPE_MASTER = "~/Flocker.rb"
+
+
+def makeHomebrewRecipeCreationFactory():
+    factory = getFlockerFactory(python="python2.7")
+    factory.addStep(SetPropertyFromCommand(
+        command=["python", "setup.py", "--version"],
+        name='check-version',
+        description=['checking', 'version'],
+        descriptionDone=['checking', 'version'],
+        property='version'
+    ))
+
+    # Run admin/homebrew.py with BuildBot sdist URL as argument
+    # XXX - how do we know the BuildBot master URL?
+    #
+    # XXX - version of make-homebrew-recipe that can handle URL argument
+    # XXX - is in ClusterHQ/flocker PR #1192
+    factory.addStep(ShellCommand(
+        name='make-homebrew-recipe',
+        description=["building", "recipe"],
+        descriptionDone=["build", "recipe"],
+        command=[
+            "VERSION=1", "admin/make-homebrew-recipe", "some-URL", ">",
+            "Flocker1.rb"],
+        haltOnFailure=True))
+
+    # Upload new .rb file to BuildBot master
+    factory.addStep(FileUpload(
+        slavesrc="Flocker1.rb",
+        masterdest=Interpolate("~/Flocker%(prop:buildnumber)s.rb")))
+
+    # Trigger the homebrew-test build
+    factory.addStep(Trigger(
+        name='trigger-homebrew-test',
+        schedulerNames=['trigger/homebrew-test'],
+        set_properties={
+            'master_recipe': Interpolate("~/Flocker%(prop:buildnumber)s.rb")
+            },
+        waitForFinish=False,
+        ))
+
+    # XXX - maybe the above should be waitForFinish=True, and then this
+    # XXX - build deletes the sdist and Homebrew files on master?
```

Comment:
If you use `resultPath` and `resultURL` consistently, they can just stick around as build artifacts.